src/main/java/com/eppo/sdk/dto/AttributeCoefficients.java

```diff
@@ -0,0 +1,7 @@
+package com.eppo.sdk.dto;
+
+public interface AttributeCoefficients {
```

Comment:
I know we have another interface in here with the `I` prefix, but I'm going to stick with the Java way of doing things where we don't do that (e.g., `Map`)